src/control/cmd/daos_server/README.md

```diff
@@ -0,0 +1,67 @@
+# DAOS Server
+
+The DAOS Server can be run either as a foreground process to issue hardware
+provisioning commands or as a background process (binary name followed by the
+"start" subcommand) when it will listen for instructions from the [management
+tool](/src/control/cmd/dmg/README.md) and manage bring-up of the DAOS system.
+A DAOS Server instance will typically be run on every storage node to
+orchestrate the operation of the managed data plane processes.
+
+## Configuration
+
+A server configuration file which is passed on invokation contains details of
+global and per-io-server parameters related to hardware devices and
+environments.
```

Comment:
Might be good to detail some of the env variables that can be set, or if there are any assumptions. Default location of the config file is also good to note.